src/entities/post.ts

```diff
@@ -0,0 +1,9 @@
+export type Post = {
```

Comment:
The indentation is correct?

Reply:
I'm not sure I understand the question. What do you feel is wrong about the identation?

Reply:
Never mind, probably GitHub doing his things 👍🏻